### PR TITLE
Preparing Crayons to be a gem

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

This PR is another attempt to help Crayons become standalone gem soon. It also provides bunch of classes missing for https://github.com/forem/forem-app/pull/6

This PR should not affect visually any views - it's just adding few classes and renaming others.

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/assets/stylesheets/base/main.scss [hunk 1]
```diff
@@ -31,3 +31,8 @@ body {
   visibility: visible;
   font-size: 18px;
 }
+
+a {
+  color: var(--link-brand-color);
+  text-decoration: none;
+}
```
Comment (vaidehijoshi) [inline app/assets/stylesheets/base/main.scss:38]: Just out of curiosity, how come we had to move this from `/scaffolds.scss` into `/base/main.scss`? 👀

**Review by vaidehijoshi — APPROVED** [review]
I did a tiny bit of QA and everything on my local instance looks good!

File: app/assets/stylesheets/base/main.scss (hunk 1 above)
Comment (ludwiczakpawel) [reply app/assets/stylesheets/base/main.scss:38]: So `/app/assets/stylesheets/*.scss` files are not part of Crayons. What is Crayons:

- `/app/assets/stylesheets/crayons.scss`
- `/app/assets/stylesheets/base/*`
- `/app/assets/stylesheets/components/*`
- `/app/assets/stylesheets/config/*`

Therefore that link styling wasn't really part of Crayons (likely one of the many missed things while building out crayons :D) so I had to move it.

Comment (vaidehijoshi) [reply app/assets/stylesheets/base/main.scss:38]: Ah, that's good to know! Thanks for explaining, @ludwiczakpawel  🙏

**Review by benhalpern — APPROVED** [review]

**Pull request merged**